scripts/gt_concordance.py

```diff
@@ -0,0 +1,551 @@
+import sys
+import os
+import re
+import subprocess
+import shutil
+
+import click
+import yaml
+import vcf
+import pyexcel_xlsx
+
+from ngi_pipeline.database.classes import CharonSession, CharonError
+from ngi_pipeline.log.loggers import minimal_logger
+from ngi_pipeline.utils.classes import with_ngi_config
+
+log = minimal_logger(os.path.basename(__file__))
+
+@click.group()
+@with_ngi_config
+@click.pass_context
+@click.option('--config', '-c', 'custom_config', help='Path to a config file', type=click.Path())
```

Comment:
this is the same config file used when running ngi_pipeline, so better you default to that value using the $NGI_CONFIG  variable